frontend/src/components/auth/LoginComponent.tsx

```diff
@@ -63,6 +63,25 @@ const LoginComponent = ({
     return false;
   };
 
+  useEffect(() => {
+    window.addEventListener("keyup", (event) => {
+      const submitButton = document.getElementById("submit-login-form");
+      if (event.key === "Enter" && submitButton !== null) {
+        if (
+          !isSignup ||
```

Comment:
hmmmmm email and password should be entered, hows `!isSignup && email !== "" && password !== ""`

Reply:
Thanks for the reviews!
To avoid checking `email !== ""` and `password !== ""` twice, I used:
```
          email !== "" &&
          password !== "" &&
          (!isSignup ||
            (firstName !== "" &&
              lastName !== "" &&
              agreeToTerms &&
              checkValidPassword()))
```
And should I add the condition `email !== "" && password !== ""` for the login button to be enabled?